travel_platform/travel_app/views.py

```diff
@@ -89,3 +90,39 @@ def get_available_cities_from_izi_travel(request):
     cities = client.search_city_by_name(city)
     return JsonResponse(cities, safe=False)
 # TODO: try to use different api to get list of cities(google for ex)
+
+
+def museums_data_for_current_city(request, venue_type):
+    city = request.GET['search-city']
+    museums = find_museums(city)
+    return render(request, 'travel_app/museums.html', {
+        "museums": museums,
+        "url_part":
+            "/travel_app/{venue_type}/search?search-city=".format(
+                venue_type=venue_type),
+        "city": city
+    })
+
+
+def attractions_data_for_current_city(request, venue_type):
+    city = request.GET['search-city']
+    tours = find_city_tours(city)
+    return render(request, 'travel_app/attractions.html', {
+        "url_part": "/travel_app/{venue_type}/search?search-city=".format(
+            venue_type=venue_type),
+        "tours": tours, "city": city
+    })
+
+
+def foursquare_venue_for_current_city(request, venue_type):
+    city = request.GET['search-city']
+    try:
+        foursquare_venues = foursquare_explore_venues(city, query=venue_type)
+    except FailedGeocode:
+        foursquare_venues = []
```

Comment:
It would be great to use [Logging](https://docs.python.org/2/library/logging.html) here. It helps to understand better why there is no records in response.